app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+# Helper methods for session handling
+module SessionsHelper
+  def logged_in?
+    !current_user.nil?
+  end
+
+  def current_user
```

Comment:
This method involves communication with the database and it should be in the Application controller or in the module you create in the lib folder. If you want to access theses methods in the view, this link might be helpful: https://apidock.com/rails/AbstractController/Helpers/ClassMethods/helper_method

Reply:
I declared it as a helper method, because it is needed in the view